time stamp
string CoordinateFormat
float64 Longitude
float64 Latitude
float64 Altitude
string latitude_label
string longitude_label
string UTMZone

